test/dynamo/test_utils.py

```diff
@@ -443,6 +445,33 @@ def test_dynamo_timed(self, mock_time, mock_time_ns):
  'triton_version': None}""",  # noqa: B950
         )
 
+    @dynamo_config.patch(
+        {
+            "log_compilation_metrics": True,
+        }
+    )
+    def test_ir_count(self):
+        def test1(x):
+            y = x + x
+            z = y * y
+            return z
+
+        compilation_events = []
+        with mock.patch("torch._dynamo.utils.log_compilation_event") as log_event:
+            torch.compile(test1)(torch.randn(10, 10))
+            compilation_events = [arg[0][0] for arg in log_event.call_args_list]
+        self.assertEqual(compilation_events[0].ir_count, 10)
```

Comment:
I don't believe this will be stable across python versions, can we make sure these tests don't fail on different Python versions?

Reply:
Yeah this isn't stable, and in fact py3.13 is failing on the PR (I tested with 3.10). Let me make this test more resilient. (I'll make it fail on unknown python versions so we update it + hardcode existing versions).

Reply:
Updated to explicitly encode versions, let me know if that's too explicit, ad we shoudl just check - ir >0 and second < first.